### PR TITLE
docs: typo in description of `Toggle`

### PR DESCRIPTION
**Pull request by UHAsikakutou**
## Description

I have corrected a typo in the description in `Toggle`.
`Toggle`コンポーネントの説明文にあった誤字を修正しました。

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: .changeset/dirty-dragons-tease.md
Comment (hirotomoyamada) [inline .changeset/dirty-dragons-tease.md:1]: Hi, this file is not needed. Please delete it.

Comment (UHAsikakutou) [reply .changeset/dirty-dragons-tease.md:1]: Thank you for your review.
I have delete it.

**Review by odendayoko — CHANGES_REQUESTED** [review]
@UHAsikakutou 
Hi, thanks for the PR.
I apologize for the inconvenience, but could you please make two additional changes?

**Comment by UHAsikakutou** [comment]
Hello, @odendayoko . 
Thank you for your review.
I have corrected the points you mentioned.

**Review by odendayoko — APPROVED** [review]
@UHAsikakutou 
LGTM Thank you!!

**Comment by hirotomoyamada** [comment]
Thanks!

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**